### PR TITLE
Added missing attribute to Result serialization

### PR DESCRIPTION
**Pull request by jbobba**

### CODE REVIEW

File: src/ngraph/serializer.cpp [hunk 1]
```diff
@@ -1407,7 +1407,10 @@ static shared_ptr<ngraph::Function>
             }
             case OP_TYPEID::Result:
             {
-                node = make_shared<op::Result>(args[0]);
+                auto needs_default_layout = node_js["needs_default_layout"].empty()
```
Comment (rkimballn1) [inline src/ngraph/serializer.cpp]: use `get_default_value<bool>(node_js, "needs_default_value", false)`

Comment (jbobba) [reply src/ngraph/serializer.cpp]: do you want me to  implement get_default_value?

Comment (rkimballn1) [reply src/ngraph/serializer.cpp]: Sorry, my mistake. The function is get_or_default, not get_default_value.

Comment (jbobba) [reply src/ngraph/serializer.cpp]: oh, nice. Didn't notice it before.

**Pull request merged**